.github/codeql-config.yml

```diff
@@ -0,0 +1,8 @@
+paths-ignore:
```

Comment:
shouldn't this go under [`on.push.paths-ignore`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) of the `codeql-code-scanning.yml` action?

Reply:
This particular setting tells the CodeQL analysis knows which files not to analyse. This means that regardless of when the analysis runs (e.g. on a PR, or on a push, or on schedule), CodeQL won't bother analysing these files or flagging up any alerts for these paths. I've based that on [the existing `.lgtm.yml` configuration file in the repository](https://github.com/mrdoob/three.js/blob/5368e6a206eba8afd93760a4b8c815e13a43c257/.lgtm.yml).

The [on.push.paths-ignore](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) is a setting that stops an Action (like the CodeQL analysis) from being triggered if changes to a _particular_ path are _pushed_. It can be useful to apply that in a CodeQL context, but it would mean that the CodeQL analysis would still flag up alerts when it is triggered by, for example, a PR being opened.